scripts/polyglot.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+${POLYGLOT_BIN}  \
+  --command=call \
+  --full_method=promlts.Store/Series \
+  --endpoint="[::]:19090" \
+  --proto_discovery_root=./pkg/store/storepb \
+  --add_protoc_includes=/Users/fabxc/repos/src/github.com/gogo/protobuf/gogoproto,/Users/fabxc/repos/src/github.com/gogo/protobuf,/Users/fabxc/repos/src/github.com/gogo/protobuf/protobuf \
```

Comment:
Worth to add example request somewhere

Reply:
Ops, that was actually not supposed to be included at all :)